### ISSUE TITLE
Add Humidity Display

### ISSUE THREAD

**Issue by cweinhofer** [balassy/MMM-RemoteTemperature#3]
Now that we have the FA5 issues taken care of, there were two other things I have been meaning to ask you about. Here's the first one (I'll post the second in a separate issue). I don't know how much time you have to work on this project, so feel free to close this if it isn't something you have the time for.

The sensor I use to send data to tmy MMM-RemoteTemperature module is both a temperature and humidity sensor. In order to display both on my MM I've jerry-rigged a solution by creating a second instance of the module to display the humidity and then used CSS to put the two together in a group. (I'm happy to send photos and/or code if that helps.)

Would it be difficult for you to add the option for the module to a humidity reading (in the same say it now supports an optional time)? I ask this knowing very little about javascript coding.

Given that dual temperature-humidity sensors are quite common, it could be a very helpful feature for people if the module supported both readings.

**Comment by balassy** [comment]
Actually it is not that difficult to implement. When I first created this module I was also thinking about it, but I did not think that people really care about humidity. Now that I have a sensor that measures humidity as well, I often found myself checking the measured value.

So yes, I will add it, and I believe I can do that in a backward compatible way.

If I would change the expected POST payload to this format, would that work with your sensor?
```
{
  "temp": 27,
  "humidity": 40.3,
  "sensorId": "1"
}
```
